test/Microsoft.IdentityModel.TestUtils/References.cs

```diff
@@ -369,6 +369,88 @@ public static byte[] T
+                return new byte[] { 227, 197, 117, 252, 2, 219, 233, 68, 180, 225, 77, 219 };
+            }
+        }
+
+        public static byte[] P
+        {
+            get
+            {
+                return new byte[] { 84, 104, 101, 32, 116, 114, 117, 101, 32, 115, 105, 103, 110, 32,
+                                    111, 102, 32, 105, 110, 116, 101, 108, 108, 105, 103, 101, 110, 99,
+                                    101, 32, 105, 115, 32, 110, 111, 116, 32, 107, 110, 111, 119, 108,
+                                    101, 100, 103, 101, 32, 98, 117, 116, 32, 105, 109, 97, 103, 105,
+                                    110, 97, 116, 105, 111, 110, 46 };
+            }
+        }
+
+        public static byte[] A
+        {
+            get
+            {
+                return new byte[] { 101, 121, 74, 104, 98, 71, 99, 105, 79, 105, 74, 83, 85, 48, 69,
+                                    116, 84, 48, 70, 70, 85, 67, 73, 115, 73, 109, 86, 117, 89, 121, 73,
+                                    54, 73, 107, 69, 121, 78, 84, 90, 72, 81, 48, 48, 105, 102, 81 };
+            }
+        }
+
+        public static byte[] E
+        {
+            get
+            {
+                return new byte[] { 229, 236, 166, 241, 53, 191, 115, 196, 174, 43, 73, 109, 39, 122,
+                                    233, 96, 140, 206, 120, 52, 51, 237, 48, 11, 190, 219, 186, 80, 111,
+                                    104, 50, 142, 47, 167, 59, 61, 181, 127, 196, 21, 40, 82, 242, 32,
+                                    123, 143, 168, 226, 73, 216, 176, 144, 138, 247, 106, 60, 16, 205,
+                                    160, 109, 64, 63, 192 };
+            }
+        }
+
+        public static byte[] T
+        {
+            get
+            {
+                return new byte[] { 92, 80, 104, 49, 133, 25, 161, 215, 173, 101, 219, 211, 136, 91,
+                                    210, 145 };
+            }
+        }
+
+        public static string EncodedAuthenticationTag
+        {
+            get { return "XFBoMYUZodetZdvTiFvSkQ"; }
+        }
+
+        public static string EncodedCipherText
+        {
+            get { return "5eym8TW_c8SuK0ltJ3rpYIzOeDQz7TALvtu6UG9oMo4vpzs9tX_EFShS8iB7j6jiSdiwkIr3ajwQzaBtQD_A"; }
+        }
+
+        public static string EncodedIV
+        {
+            get { return "48V1_ALb6US04U3b"; }
+        }
+
+        public static string Result
+        {
+            get { return "eyJhbGciOiJSU0EtT0FFUCIsImVuYyI6IkEyNTZHQ00ifQ.OKOawDo13gRp2ojaHV7LFpZcgV7T6DVZKTyKOMTYUmKoTCVJRgckCL9kiMT03JGeipsEdY3mx_etLbbWSrFr05kLzcSr4qKAq7YN7e9jwQRb23nfa6c9d-StnImGyFDbSv04uVuxIp5Zms1gNxKKK2Da14B8S4rzVRltdYwam_lDp5XnZAYpQdb76FdIKLaVmqgfwX7XWRxv2322i - vDxRfqNzo_tETKzpVLzfiwQyeyPGLBIO56YJ7eObdv0je81860ppamavo35UgoRdbYaBcoh9QcfylQr66oc6vFWXRcZ_ZT2LawVCWTIy3brGPi6UklfCpIMfIjf7iGdXKHzg.48V1_ALb6US04U3b.5eym8TW_c8SuK0ltJ3rpYIzOeDQz7TALvtu6UG9oMo4vpzs9tX_EFShS8iB7j6jiSdiwkIr3ajwQzaBtQD_A.XFBoMYUZodetZdvTiFvSkQ"; }
```

Comment:
I'm a bit skeptical about the spacing in this string. Is it used anywhere?
.."322i - vDx"...

Reply:
It is not used. I've corrected the spacing(not sure how they were added in the first place).